src/nav/full-stack-observability.yml

```diff
@@ -2583,13 +2583,15 @@ pages:
           - title: Enable Lambda monitoring
             path: /docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring
             pages:
-              - title: 1. Account linking
+              - title: Intro to enabling Lambda monitoring
+                path: /docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/enable-aws-lambda-monitoring
+              - title: Step 1. Account linking
                 path: /docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/account-linking
-              - title: 2. Instrumentation examples
+              - title: Step 2. Instrumentation examples
                 path: /docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/instrument-example
-              - title: 3. Instrument your functions
+              - title: Step 3. Instrument your functions
                 path: /docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/instrument-your-own
-              - title: 4. Next steps
+              - title: Step 4. Next steps
```

Comment:
@zuluecho9, the use of "steps" conflicts with the existing steps. Since this document is all about configuration, how about changing it to "Step 4: Configuration"?